lib/msf/ui/console/stylers/rank_styler.rb

```diff
@@ -0,0 +1,17 @@
+
+class RankStyler
+  include Msf
+
+  def style(rank)
+    case rank
+    when RankingName[GoodRanking]
+      "%blu#{rank}%clr"
+    when RankingName[GreatRanking]
+      "%yel#{rank}%clr"
+    when RankingName[ExcellentRanking]
+      "%grn#{rank}%clr"
```

Comment:
Chosen to minimize negative effects for color blind users based off this [chart](https://usabilla.com/blog/wp-content/uploads/2015/11/colorblind_01_03-1.jpg). If anyone is/knows someone with color-blindness I'd appreciate their input on how easy the below is to read:

Example output for `search sql`
![Screenshot 2020-03-13 at 14 39 52](https://user-images.githubusercontent.com/54621924/76630802-8b6d6e00-6538-11ea-95d9-df032afb7895.png)

Example output for `search exploit/windows/email`:
![Screenshot 2020-03-13 at 14 41 37](https://user-images.githubusercontent.com/54621924/76630954-c40d4780-6538-11ea-8252-85d5c6bc8daa.png)

Reply:
just my two cents but I don't like it, still pretty noisy, I'm not UX...but I think if we just say anything good or above is green that'd be a clean look

Reply:
🌈

Reply:
I agree 👍

I think it'd be good to keep this simple, just highlighting matched search strings and and using green for excellent/great

Reply:
Votes are in and `Little to No Unicorn Vomit` has won! I'll pull out `Check` highlighting and restrict  the `Rank` coloring to Green for excellent/good

Reply:
New more minimal color scheme, looks way better:
![Screenshot 2020-03-13 at 15 42 35](https://user-images.githubusercontent.com/54621924/76636886-b27c6d80-6541-11ea-9296-774fe2ef062b.png)
![Screenshot 2020-03-13 at 15 43 07](https://user-images.githubusercontent.com/54621924/76636902-b6a88b00-6541-11ea-9236-cb275edbd304.png)

Reply:
I love it.